src/server/worker/pipeline/transform/transform_test.go

```diff
@@ -77,42 +69,45 @@ func newWorkerSpawnerPair(t *testing.T, dbConfig serviceenv.ConfigOption, pipeli
 		},
 	})
 	require.NoError(t, err)
+	branchInfo, err := env.PachClient.PfsAPIClient.InspectBranch(ctx, &pfs.InspectBranchRequest{Branch: pipelineRepo.NewBranch(pipelineInfo.Details.OutputBranch)})
+	require.NoError(t, err)
+	_, err = env.PachClient.PfsAPIClient.FinishCommit(ctx, &pfs.FinishCommitRequest{Commit: branchInfo.Head, Force: true})
+	require.NoError(t, err)
 
 	// Create the meta system repo and set up the branch provenance
 	metaRepo := client.NewSystemRepo(pipelineInfo.Pipeline.Name, pfs.MetaRepoType)
 	_, err = env.PachClient.PfsAPIClient.CreateRepo(ctx, &pfs.CreateRepoRequest{Repo: metaRepo})
 	require.NoError(t, err)
 	_, err = env.PachClient.PfsAPIClient.CreateBranch(ctx, &pfs.CreateBranchRequest{
-		Branch:     metaRepo.NewBranch("master"),
-		Provenance: []*pfs.Branch{pipelineRepo.NewBranch("master")},
+		Branch: metaRepo.NewBranch("master"),
+		Provenance: []*pfs.Branch{
+			pipelineRepo.NewBranch(pipelineInfo.Details.OutputBranch),
+		},
 	})
 	require.NoError(t, err)
+	branchInfo, err = env.PachClient.PfsAPIClient.InspectBranch(ctx, &pfs.InspectBranchRequest{Branch: metaRepo.NewBranch("master")})
+	require.NoError(t, err)
+	_, err = env.PachClient.PfsAPIClient.FinishCommit(ctx, &pfs.FinishCommitRequest{Commit: branchInfo.Head, Force: true, Error: "force close"})
```

Comment:
here too, or just add to the message (force close so it's skipped by the worker code)